source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -451,7 +453,7 @@ class VehicleJourneySerializer(GenericSerializer):
     stop_times = StopTimeSerializer(many=True)
     comments = CommentSerializer(many=True, display_none=False)
     comment = FirstCommentField(attr='comments', display_none=False)
-    codes = CodeSerializer(many=True)
+    codes = CodeSerializer(many=True, required=False)
```

Comment:
why `codes` are involved?

Reply:
Because of the schema check. We don't send "codes" for a vehicle journeys... The new tests highlight the problem.

Reply:
ok! thx